packages/react-core/src/components/Progress/ProgressHelperText.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import progressStyle from '@patternfly/react-styles/css/components/Progress/progress';
+import { HelperText, HelperTextItem, HelperTextItemProps } from '../HelperText';
+import { ProgressProps } from './Progress';
+
+export interface ProgressHelperTextProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the progress helper text component. */
+  children?: React.ReactNode;
+  /** Variant of the progress component. */
+  progressVariant?: ProgressProps['variant'];
+}
+
+export const ProgressHelperText: React.FunctionComponent<ProgressHelperTextProps> = ({
+  children,
+  progressVariant
```

Comment:
I wouldn't be opposed to doing so. I originally had spread props but I removed them because it didn't feel like this component made sense to expose publicly, and given our current use case they didn't seem to add any value.